### PR TITLE
GCE provider: add wrapper for security policy

### PR DESCRIPTION
**Pull request by MrHohn**
**What this PR does / why we need it**:
Generated mock interface for beta backend service and security policy as well.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #NONE 

**Special notes for your reviewer**:
/assign @nicksardo   

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by MrHohn** [comment]
@nicksardo Sorry, `gen_test.go` is updated now.

File: pkg/cloudprovider/providers/gce/cloud/meta/meta.go [hunk 1]
```diff
@@ -327,6 +338,14 @@ var AllServices = []*ServiceInfo{
 		keyType:     Global,
 		serviceType: reflect.TypeOf(&ga.RoutesService{}),
 	},
+	{
+		Object:      "SecurityPolicy",
+		Service:     "SecurityPolicies",
+		Resource:    "securitypolicies",
```
Comment (nicksardo) [inline pkg/cloudprovider/providers/gce/cloud/meta/meta.go]: `securityPolicies`

Comment (MrHohn) [reply pkg/cloudprovider/providers/gce/cloud/meta/meta.go]: Thanks, fixed.

**Comment by MrHohn** [comment]
Added a second commit to point to `hack/update-cloudprovider-gce.sh` in doc.

**Pull request merged**